documentation/ansible-playbook/modules/ansible-playbook-contents-of-the-webserver.adoc

```diff
@@ -57,20 +57,20 @@ NOTE: The following manual procedure can be skipped if used the <<helper_script,
 
 .Manual Procedure
 //FIXME This link doesn't exist bc ocp4.5 isn't GA, should we just point to 4.4 as an example? Do we want to provide the dev link too?
-. Download the {product-title} version {release} latest https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-{release}/release.txt[release.txt] file
+. Download the {product-title} version {product-version} latest https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-{product-version}/release.txt[release.txt] file
 +
 [source,bash]
 [subs="attributes"]
 ----
 [user@webserver ~]$ cd /path/to/webserver/dir
-[user@webserver ~]$ wget https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-{release}/release.txt
+[user@webserver ~]$ wget https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-{product-version}/release.txt
 ----
 +
 [NOTE]
 ====
 When working with a development version of {product-tile}, use the following link for the
```

Comment:
Should that be {product-title}?

Reply:
wasn't the var going to be product-version?

Reply:
I saw the tipo, on line 71, the other one is ok and gets rendered properly